### PR TITLE
update coveralls

### LINKED ISSUE #363: Add coveralls to this repo
There is a reference to a coveralls reporter in the [package.json `travis:karma` task](https://github.com/mozilla/activity-streams/blob/52ab9d4c066a5df441ae7b9c6e56e6f0ff569465/package.json#L115):

```js
"travis:karma": "NODE_ENV=test karma start --reporters mocha,coverage,coveralls",
```

I can't seem to find any code coverage numbers for this repo anywhere (like the [Travis-CI logs](https://travis-ci.org/mozilla/activity-streams/builds/118106067#L1666)), or on [coveralls.io](https://coveralls.io/github/mozilla/activity-streams) dashboard page.

We should probably get this wired up, so we're confident in our code coverage that we're in the 90%+ percentile (as opposed to the 35%).

### PR DESCRIPTION
**Pull request by k88hudson**
Fixes #363

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/activity-streams/551)

### CODE REVIEW

**Comment by k88hudson** [comment]
@pdehaan R?

File: .travis.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ cache:
     - node_modules
 
 addons:
-  firefox: latest-beta
+  firefox: "latest-beta"
```
Comment (k88hudson) [inline .travis.yml]: I tried changing this to 35, but it fails for some reason...

Comment (k88hudson) [inline .travis.yml]: Filed https://github.com/mozilla/activity-streams/issues/552

**Comment by pdehaan** [comment]
91% coverage!! Nice!
Bonus points if we add the badge to the README to keep us honest.

:+1:

**Comment by pdehaan** [comment]
@k88hudson This doesn't address anything in #525, correct?

**Comment by k88hudson** [comment]
@pdehaan it does (partially) since it fixes the issue where travis is reporting stuff, although the problem could still happen if people run `jpm test` instead of `npm test`

**Pull request merged**